src/basic/fd-util.h

```diff
@@ -101,6 +102,9 @@ int read_nr_open(void);
 int fd_get_diskseq(int fd, uint64_t *ret);
 
 int dir_fd_is_root(int dir_fd);
+static inline int dir_fd_is_root_or_cwd(int dir_fd) {
+        return dir_fd == AT_FDCWD ? true : dir_fd_is_root(dir_fd);
```

Comment:
Ternary operator is not really needed here?

Reply:
> Ternary operator is not really needed here?

Hm? What do you mean? Note, dropping `true` triggers warning:
```
../src/basic/fd-util.h: In function ‘dir_fd_is_root_or_cwd’:
../src/basic/fd-util.h:106:36: warning: the omitted middle operand in ‘?:’ will always be ‘true’, suggest explicit middle operand [-Wparentheses]
  106 |         return dir_fd == AT_FDCWD ?: dir_fd_is_root(dir_fd);
      |                                    ^
```

Reply:
`dir_fd == AT_FDCWD || dir_fd_is_root(dir_fd)` maybe?

Reply:
Oh, never mind. `dir_fd_is_root` can return negative errno.,,

Sorry for the noise.

Reply:
No problem. Thank you for your review!